tests/data/src/prjwithdatafile/setup.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages
+
+setup(
+    name='pip_issue_4092',
+    version="1.0",
+    packages=['pip_issue_4092'],
+    data_files=[
+        (r'packages1', ['pip_issue_4092/README.txt']),
+        (r'packages2', ['pip_issue_4092/README.txt'])
+    ]
+)

```

Comment:
Ideally, the whole package (and its wheel) should be named `prjwithdatafile`. The goal being to easily spot the source of the `pip_issue_4092-1.0-py2.py3-none-any.whl` file.